src/sdks/tableau/methods/serverMethods.ts

```diff
@@ -29,6 +30,20 @@ export default class ServerMethods extends AuthenticatedMethods<typeof serverApi
   getServerInfo = async (): Promise<ServerInfo> => {
```

Comment:
I accidentally added this as an authenticated method initially. It does not require an access token.